api/v1alpha1/remoteistio_types.go

```diff
@@ -0,0 +1,258 @@
+// +kubebuilder:validation:XValidation:rule="!has(self.values) || !has(self.values.global) || !has(self.values.global.istioNamespace) || self.values.global.istioNamespace == self.__namespace__",message="spec.values.global.istioNamespace must match spec.namespace"
+type RemoteIstioSpec struct {
+	// +sail:version
+	// Defines the version of Istio to install.
+	// Must be one of: v1.21.0, latest.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=1,displayName="Istio Version",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldGroup:General", "urn:alm:descriptor:com.tectonic.ui:select:v1.21.0", "urn:alm:descriptor:com.tectonic.ui:select:latest"}
+	// +kubebuilder:validation:Enum=v1.21.0;latest
+	// +kubebuilder:default=v1.21.0
+	Version string `json:"version"`
+
+	// Defines the update strategy to use when the version in the RemoteIstio CR is updated.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Update Strategy"
+	// +kubebuilder:default={type: "InPlace"}
+	UpdateStrategy *RemoteIstioUpdateStrategy `json:"updateStrategy,omitempty"`
+
+	// +sail:profile
+	// The built-in installation configuration profile to use.
+	// The 'default' profile is always applied. On OpenShift, the 'openshift' profile is also applied on top of 'default'.
+	// Must be one of: ambient, default, demo, empty, external, minimal, openshift-ambient, openshift, preview, remote, stable.
+	// +++PROFILES-DROPDOWN-HIDDEN-UNTIL-WE-FULLY-IMPLEMENT-THEM+++operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Profile",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldGroup:General", "urn:alm:descriptor:com.tectonic.ui:select:ambient", "urn:alm:descriptor:com.tectonic.ui:select:default", "urn:alm:descriptor:com.tectonic.ui:select:demo", "urn:alm:descriptor:com.tectonic.ui:select:empty", "urn:alm:descriptor:com.tectonic.ui:select:external", "urn:alm:descriptor:com.tectonic.ui:select:minimal", "urn:alm:descriptor:com.tectonic.ui:select:preview", "urn:alm:descriptor:com.tectonic.ui:select:remote"}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
+	// +kubebuilder:validation:Enum=ambient;default;demo;empty;external;minimal;openshift-ambient;openshift;preview;remote;stable
+	Profile string `json:"profile,omitempty"`
+
+	// Namespace to which the Istio components should be installed.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Namespace"}
+	// +kubebuilder:default=istio-system
+	Namespace string `json:"namespace"`
+
+	// Defines the values to be passed to the Helm charts when installing Istio.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Helm Values"
+	Values *Values `json:"values,omitempty"`
+}
+
+// RemoteIstioUpdateStrategy defines how the control plane should be updated when the version in
+// the RemoteIstio CR is updated.
+type RemoteIstioUpdateStrategy struct {
```

Comment:
I think I initially created this, because I was expecting it to diverge from IstioUpdateStrategy. But I think you're right that we can use a single struct.